client/main.js

```diff
@@ -106,9 +106,9 @@ $(document).ready(function() {
     // About
     window.location = '/about';
   });
-  window.Mousetrap.bind('g n s', () => {
```

Comment:
@dhcodes what, Stuart means I believe that this file `client/main.js` is going to be ignored with regards to the keyboard shortcuts. 

In any case this cleanup is probably good to do, to avoid confusion.